h2o-algos/src/main/java/hex/rulefit/RuleFit.java

```diff
@@ -272,8 +274,10 @@ public void computeImpl() {
                 // TODO: add here coverage_count and coverage percent
                 model._output._rule_importance = convertRulesToTable(getRules(glmModel.coefficients(), ruleEnsemble));
                 
-                model._output._model_summary = generateSummary(glmModel, ruleEnsemble.size(), overallTreeStats, ntrees);
-
+                model._output._model_summary = generateSummary(glmModel, ruleEnsemble != null ? ruleEnsemble.size() : 0, overallTreeStats, ntrees);
+                
+                model.dataFromRulesCodes = dataFromRulesCodes;
```

Comment:
interesting - why is this persisted in the model instead of the model._output - I wonder if there is a reason for that

Reply:
I just thought that it has no special added value (apart of mojo) to be in output, so I put it on the model, but I can put  it into output if it's more suitable to have it there

Reply:
yes, better to have it in the output